dfx/index.ts

```diff
@@ -92,3 +127,48 @@ export async function getIdentity(
     );
     return Secp256k1KeyIdentity.fromPem(await readFile(identityPath, 'utf-8'));
 }
+
+export function getPemKey(identityName: string = getIdentityName()): string {
```

Comment:
These defaults are scary, I think we should not use them and just explicitly pass in our parameters considering the absolutely intense nature of getting this identity stuff correct

Reply:
sure